### ISSUE TITLE
Outline shader fill transparency

### ISSUE THREAD

**Issue by SoulMuncher** [jdoleary/Spellmasons#339]
Outlines affect the color of ghost archers, blood trails, and potentially other entities.

**Comment by jdoleary** [comment]
![image](https://github.com/jdoleary/Spellmasons/assets/6413317/ce5fd1e6-f7cc-4222-bdec-14d2256715a0)
Color looks fine to me.

Also outlines are for accessibility so if there's minor visual aberrations I'm not worried about it.  It's to help players recognize / distinguish enemies from the environment and allies.

**Comment by SoulMuncher** [comment]
It's very slight, but enough for me to feel like something was "off" and notice that the game was unusually "dreary looking", before I realized exactly what was happening.

Heres the video comparison I sent on discord the other day:

https://github.com/jdoleary/Spellmasons/assets/68413161/0b652c11-f34b-4a77-98d7-b132432bad2f

**Comment by jdoleary** [comment]
I see the difference, but it's acceptable.

**Comment by SoulMuncher** [comment]
This would explain why it only seems to affect transparent sprites:

https://stackoverflow.com/questions/72056609/pixi-js-how-to-draw-outline-of-container-while-keeping-its-content-transparent

**Comment by jdoleary** [comment]
@SoulMuncher If you disagree with a judgement call I've made, we can talk about it; but do not undo actions that I've taken.  This makes me feel disrespected.

**Comment by SoulMuncher** [comment]
@jdoleary 
I'm sorry. It wasn't my intention to make you feel disrespected 😓 I re-opened the issue because I found what might be a quick and easy solution for you to consider. I'll make sure to run any un-doing by you in the future beforehand.

Though, I don't think we should close bugs as "not planned" unless we've determined it's not a bug and is 100% working as intended. This is not working as intended and could lead to more significant visual artifacts in the future. The isn't the worst instance of the bug either, it's just the first recording I had. It seems like a better idea to keep it tracked and marked as low prio than to say it's not that bad and pretend it doesn't exist. Maybe we'll get back around to fixing it, maybe a community member will bring up a related report that makes it higher prio, but ultimately it's still an unresolved issue and should be kept track of imo
